examples/cluster/plot_cluster_comparison.py

```diff
@@ -3,24 +3,23 @@
 Comparing different clustering algorithms on toy datasets
 =========================================================
 
-This example shows characteristics of different
-clustering algorithms on datasets that are "interesting"
-but still in 2D. With the exception of the last dataset,
-the parameters of each of these dataset-algorithm pairs
-has been tuned to produce good clustering results. Some
-algorithms are more sensitive to parameter values than
-others.
-
-The last dataset is an example of a 'null' situation for
-clustering: the data is homogeneous, and there is no good
-clustering. For this example, the null dataset uses the
-same parameters as the dataset in the row above it, which
-represents a mismatch in the parameter values and the
-data structure.
-
-While these examples give some intuition about the
-algorithms, this intuition might not apply to very high
-dimensional data.
+The following example shows the characteristic of
```

Comment:
For the text in this paragraph, could you reformat it to make sure it fits as much as possible the 80 characters.